draft-ietf-modpod-group-processes.md

```diff
@@ -289,6 +289,9 @@ the scope of this memo. Examples of actions that are out of scope include,
 but are not limited to, datatracker account removal, in-person meeting
 registration, content removal or redaction, moderation or policing of
 private or non-IETF communications, and redaction from IETF archives.
+While the moderator team does not moderate non-public IETF mailing
+lists, they can define processes that the administrators of such lists
+can choose to use.
```

Comment:
```suggestion
lists, the administrators of such lists can choose to adopt some of the
processes that the moderator team develops.
```
To make it clear the mod team is not tasked to optionally define process for non-public lists.

Reply:
Here's my issue: I don't know that the same guidance can or should apply, so you might need both.

Reply:
This is subtle. Here are the things I think we agree on:
* Moderators can't moderate private lists
* Moderators shouldn't be in the business of saying "here's how you should moderate the IESG list"
* When defining process, moderators should focus on public lists
* If moderators define a process that applies to all IETF lists, administrators of private lists can choose to use that process

The one point where there's a subtle difference between my proposed sentence and Lars's is: when the moderators are defining process, are they allowed to say "We're defining a process for all lists, but it would apply slightly differently to a private list, should those administrators choose to use it"?

My preferred answer to that question is Yes.

Reply:
@DavidSchinazi how about we leave this for the next time?